package.json

```diff
@@ -53,7 +53,8 @@
       "ts-jest": {
         "skipBabel": true
       }
-    }
+    },
+    "testURL": "http://localhost/"
```

Comment:
I saw you opened a 2nd PR for this! could you remove this change here? Thanks!

Reply:
:+1: thanks for merging that other PR, will update latest master here